FakeItEasy.sln

```diff
@@ -10,17 +10,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.mailmap = .mailmap
 		appveyor.yml = appveyor.yml
 		build.cmd = build.cmd
-		build.csx = build.csx
 		CONTRIBUTING.md = CONTRIBUTING.md
 		Directory.Build.props = Directory.Build.props
-		FakeItEasy.Dictionary.xml = FakeItEasy.Dictionary.xml
-		FakeItEasy.ruleset = FakeItEasy.ruleset
-		FakeItEasy.snk = FakeItEasy.snk

```

Comment:
Shouldn't these files be under `src` ?

Reply:
Er, yes. I forgot about the files you moved to `src`. I was only thinking of the new `tools` dir.